File: 1812-determine-color-of-a-chessboard-square/1812-determine-color-of-a-chessboard-square.js
/**
 * @param {string} coordinates
 * @return {boolean}
 */
var squareIsWhite = function(coordinates) {
    const column = coordinates.charCodeAt(0) - 96; // Transforma toda a string da coordinates em char ASCI, removendo também os 96 chars
    const row = parseInt(coordinates[1]); // Parses just the Int values

    return (column + row) % 2 !== 0;
};

File: 1812-determine-color-of-a-chessboard-square/1812-determine-color-of-a-chessboard-square.cs
public class Solution {
    public bool SquareIsWhite(string coordinates) {
        byte colByte  = (byte)(coordinates[0] - 96);
        byte rowByte = (byte)(coordinates[1] - 48);
        
        int sum = colByte + rowByte;
        return sum % 2 != 0;
        }
}